buildSrc/src/main/kotlin/splunk.java-conventions.gradle.kts

```diff
@@ -18,7 +18,7 @@ val otelAlphaVersion = "1.12.0-alpha"
 val otelContribAlphaVersion = "1.12.0-alpha"
 val otelInstrumentationVersion = "1.13.0-SNAPSHOT"
 val otelInstrumentationAlphaVersion = "1.13.0-alpha-SNAPSHOT"
-val micrometerVersion = "1.8.2"
+val micrometerVersion = "1.8.4"
```

Comment:
micrometer version is also mentioned in the readme

Reply:
Oh, nice catch, I completely forgot about it. Thanks!